sentry/src/main/java/io/sentry/logger/LoggerApi.java

```diff
@@ -211,6 +212,19 @@ private void captureLog(
           "sentry.environment",
           new SentryLogEventAttributeValue(SentryAttributeType.STRING, environment));
     }
+
+    final @Nullable SentryId replayId = scopes.getCombinedScopeView().getReplayId();
```

Comment:
so I think we can simplify this by quite a bit:
- First get the replay_id from `scopes`
  - If it's not empty that means all good and we're in session mode (or about to enter it) - hence we don't have send `sentry._internal.replay_is_buffering` at all
  - If it's empty - we try to get the replay_id from `scopes.getOptions().getReplayController().getReplayId()`
    - If it's not empty that means we're in buffer mode - hence we have to send  `sentry._internal.replay_is_buffering: true`
    - If this one is still empty - that means there's no replay being recorded right now and we just don't send any of that
    
 This way we wouldn't have to keep `replayType` in the scope either. How that sound?

Reply:
@romtsn do you like it now? 😄